packages/commonwealth/client/scripts/views/pages/create_community/starter_community_form.tsx

```diff
@@ -63,6 +63,7 @@ export const StarterCommunityForm = () => {
           { label: 'ethereum', value: 'ethereum' },
           { label: 'near', value: 'near' },
         ]}
+        initialValue={{ label: 'ethereum', value: 'ethereum' }}
```

Comment:
this was initially displaying "cosmos", but sending values for Ethereum if dropdown is unchanged.